Sources/SwiftProtobuf/Internal.swift

```diff
@@ -0,0 +1,42 @@
+// Sources/SwiftProtobuf/Message.swift - Message support
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See LICENSE.txt for license information:
+// https://github.com/apple/swift-protobuf/blob/master/LICENSE.txt
+//
+// -----------------------------------------------------------------------------
+///
+/// Internal helpers on Messages for the library. These are public
+/// just so the generated code can call them, but shouldn't be called
+/// by developers directly.
+///
+// -----------------------------------------------------------------------------
+
+
+import Foundation
+import Swift
+
+public struct Internal {
+  private init() {}
+
+  public static func areAllInitialized<M: Message>(_ listOfMessages: [M]) -> Bool {
```

Comment:
I wonder how different the performance/code size would be if we made this non-generic and the argument type `[Message]` instead (and below).

In this case, since we don't need the generics, we should probably drop them.